pyro/contrib/autoguide/__init__.py

```diff
@@ -521,15 +521,15 @@ class AutoLowRankMultivariateNormal(AutoContinuous):
         guide = AutoLowRankMultivariateNormal(model, rank=10)
         svi = SVI(model, guide, ...)
 
-    By default the ``D_term`` is initialized to 1/2 and the ``W_term`` is
-    intialized randomly such that ``W_term.matmul(W_term.t())`` is half the
+    By default the ``diagonal_term`` is initialized to 1/2 and the ``cov_factor`` is
+    intialized randomly such that ``cov_factor.matmul(cov_factor.t())`` is half the
     identity matrix. To change this default behavior the user
     should call :func:`pyro.param` before beginning inference, e.g.::
 
         latent_dim = 10
         pyro.param("auto_loc", torch.randn(latent_dim))
-        pyro.param("auto_W_term", torch.randn(latent_dim, rank)))
-        pyro.param("auto_D_term", torch.randn(latent_dim).exp()),
+        pyro.param("auto_cov_factor", torch.randn(latent_dim, rank)))
+        pyro.param("auto_cov_diagonal", torch.randn(latent_dim).exp()),
```

Comment:
It looks like these are named `cov_factor` and `cov_diag` upstream.
https://pytorch.org/docs/master/_modules/torch/distributions/lowrank_multivariate_normal.html
Can we match those as `auto_cov_factor` and `auto_cov_diag`?